R/extractManualRecode.R

```diff
@@ -1,10 +1,22 @@
 #' Get manual recodes
 #'
-#' @param recodedList A data frame with the automatically recoded values.+
+#' @param recodedList A data frame with the automatically recoded values.
```

Comment:
`@inheritParams useRecodeList` will inherit the definition for `recodedList` from `useRecodeList()`, so it stays consistent over mutliple functions

Reply:
In `useRecodeList` , the parameter is called `recodeList`, and in `extractManualRecode` it's`recodedList`. To use `@inheritParams useRecodeList`, we would need to change the parameter name.

Reply:
Ah thanks, I didn't read closely enough. In this case we need to keep it like before, because it's two distinct arguments.